Content.Server/Radiation/Components/RadiationReceiverComponent.cs

```diff
@@ -10,6 +10,13 @@ namespace Content.Server.Radiation.Components;
 [Access(typeof(RadiationSystem))]
 public sealed class RadiationReceiverComponent : Component
 {
+    /// <summary>
+    ///     Does this object can receive radiation rays?
+    ///     If false will ignore any radiation sources.
+    /// </summary>
+    [DataField("canReceive")]
+    public bool CanReceive = true;
+
```

Comment:
Toggling components off / on by a bool can be a code smell if it's something frequently being used in a subscription (i.e. the geiger, radreceiver query) when you could just add / remove the component.